docs/examples/notebooks/03-integrator.ipynb

```diff
@@ -211,17 +211,76 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "plt.figure(figsize=(8, 5))\n",
-    "plt.plot(sim.trange(), sim.data[input_p], 'k--', label=\"Input\")\n",
-    "plt.plot(sim.trange(), sim.data[output_p], label=\"Integrator output\")\n",
-    "plt.legend();"
+    "def plot_results():\n",
+    "    plt.figure(figsize=(8, 5))\n",
+    "    plt.plot(sim.trange(), sim.data[input_p], 'k--', label=\"Input\")\n",
+    "    plt.plot(sim.trange(), sim.data[output_p], label=\"Integrator output\")\n",
+    "    plot_ideal()\n",
+    "    plt.legend();\n",
+    "\n",
+    "\n",
+    "def plot_ideal():\n",
+    "    # Obtain the input function data from the input_node\n",
+    "    input_t = list(input_node.output.data.keys())\n",
+    "    input_v = list(input_node.output.data.values())\n",
+    "\n",
+    "    # Construct the ideal output (assumes 1D signal)\n",
+    "    values = [[0]]\n",
+    "    input_t += [sim.trange()[-1]]\n",
+    "    for i, v in enumerate(input_v):\n",
+    "        values += [values[-1] + v * (input_t[i + 1] - input_t[i])]\n",
+    "\n",
+    "    # Make the plot\n",
+    "    plt.plot(input_t, values, label=\"Ideal\")\n",
+    "\n",
+    "\n",
+    "plot_results()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The plot above shows the neurons effectively integrating the input signal. Because of the implementation in neurons, the integration is not perfect (i.e., there will be some drift). Run the simulation several times to get a sense of the kinds of drift you might expect."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Step 9: Spiking Neurons\n",
+    "\n",
+    "The plots above demonstrate the results of an integrator network implemented with non-spiking linear rectified neurons. The network can also be simulated using spiking neurons to illustrate the similarities and differences between a spiking and a non-spiking network.\n",
```

Comment:
I think it's more consistent to say "rectified linear" instead of "linear rectified"